sips/sip-017/sip-017-schnorr.md

```diff
@@ -0,0 +1,120 @@
+# SIP-017 — Schnorr signatures
+
+## Preamble
+
+**SIP:** 017 (awaiting acceptance)
+
+**Title:** Schnorr signatures
+
+**Author:** Asteria [asteria@syvita.org](mailto:asteria@syvita.org)
+
+**Consideration:** Technical
+
+**Type:** Consensus
+
+**Layer:** Consensus (soft fork)
+
+**Status:** Draft
+
+**Created:** 19 December 2021
+
+**License:** BSD 2-Clause
+
+**Sign-off:** —
+
+## Abstract
+
+This SIP describes the addition of Schnorr signature structure and validation to the account model for the Stacks blockchain.
+
+## Introduction
+
```

Comment:
Formatting nitpick: can you make sure all lines are about ~100 characters long, so that when someone submits a patch it doesn't create a ton of merge conflicts?